src/service/resources-impl.js

```diff
@@ -169,6 +173,7 @@ export class Resources {
     });
     this.viewport_.onScroll(() => {
       this.lastScrollTime_ = timer.now();
+      this.framerate_.collect();
```

Comment:
right, we do need to call flush at some point for these. one i'm guessing should be on doc unload and one when the queue reaches max (100 if i remember correctly)?